src/vaults/libraries/VaultPricingLib.sol

```diff
@@ -72,9 +83,18 @@ library VaultPricingLib {
         return (_value / 10 ** (PRICE_DECIMALS - decimals)).toUint128();
```

Comment:
CR: What if decimals > PRICE_DECIMALS? I think here we need to use something similar to `convertWithPrice()`.

Same in `toPriceDecimals()`

Reply:
It seems like this code was already on the main, so we can add an issue to solve it later

Reply:
Good catch. Agree!

Reply:
`decimals` can never be more than PRICE_DECIMALS. Since we limit decimals to 18 in `registerAsset`

Reply:
Maybe adding a comment indicating that is interesting (?)

Reply:
Will do!